File: CS/CustomPreviousPageCommand.cs
using System;
using System.Windows.Input;

namespace DXSample {
  public  class CustomPreviousPageCommand : ICommand {
        ICommand baseCommand;
        public CustomPreviousPageCommand(ICommand baseCommand) {
            this.baseCommand = baseCommand;
        }

        public event EventHandler CanExecuteChanged {
            add { baseCommand.CanExecuteChanged += value; }
            remove { baseCommand.CanExecuteChanged -= value; }
        }

        public bool CanExecute(object parameter) {
            return baseCommand.CanExecute(parameter);
        }

        public void Execute(object parameter) {
            baseCommand.Execute(parameter);
        }
    }
}

File: VB/CustomPreviousPageCommand.vb
Imports System
Imports System.Windows.Input

Namespace DXSample
  Public Class CustomPreviousPageCommand
      Implements ICommand

        Private baseCommand As ICommand
        Public Sub New(ByVal baseCommand As ICommand)
            Me.baseCommand = baseCommand
        End Sub

        Public Custom Event CanExecuteChanged As EventHandler Implements ICommand.CanExecuteChanged
            AddHandler(ByVal value As EventHandler)
                AddHandler baseCommand.CanExecuteChanged, value
            End AddHandler
            RemoveHandler(ByVal value As EventHandler)
                RemoveHandler baseCommand.CanExecuteChanged, value
            End RemoveHandler
            RaiseEvent(ByVal sender As System.Object, ByVal e As System.EventArgs)
            End RaiseEvent
        End Event

        Public Function CanExecute(ByVal parameter As Object) As Boolean Implements ICommand.CanExecute
            Return baseCommand.CanExecute(parameter)
        End Function

        Public Sub Execute(ByVal parameter As Object) Implements ICommand.Execute
            baseCommand.Execute(parameter)
        End Sub
  End Class
End Namespace